pkgs/applications/virtualization/gvisor/default.nix

```diff
@@ -0,0 +1,107 @@
+{ stdenv
+, buildBazelPackage
+, fetchFromGitHub
+, cacert
+
+, git
+, glibcLocales
+, go
+, iproute
+, iptables
+, procps
+, python3
+}:
+
+let
+  preBuild = ''
+    patchShebangs .
+
+    # Tell rules_go to use the Go binary found in the PATH
+    sed -E -i \
+      -e 's|go_version\s*=\s*"[^"]+",|go_version = "host",|g' \
+      WORKSPACE
+
+    # The gazelle Go tooling needs CA certs
+    export SSL_CERT_FILE="${cacert}/etc/ssl/certs/ca-bundle.crt"
```

Comment:
This is used during the fetcher phase of `buildBazelPackage`. I assume it's not baked into the binary, right?

Quickly looking through nixpkgs, it seems `pkgs/build-support/build-bazel-package/default.nix` already sets `GIT_SSL_CAINFO=`. Isn't that not enough? What do you think about setting `SSL_CERT_FILE` in there, too, instead of here?

Reply:
I could do that; opened as #74536 if that's something you'd want to merge.

The reason that `GIT_SSL_CAINFO` isn't enough is because it's not just `git` fetching things; `rules_go` runs a Go binary that fetches sources, which means that it needs to know about SSL certs.  If you're curious: the Go tooling makes a request with `?go-get=1` to see where it should fetch a source from; more details [here](https://golang.org/cmd/go/).

Reply:
Thanks for the explanation :-)

#74536 was merged - can you remove the line from here?

Reply:
@flokli - Thanks for the quick merge! Removed this line and rebased on `master`.